.ci/packer_cache.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+#
+# this file is run daily to generate worker packer images
+#
+
+source /usr/local/bin/bash_standard_lib.sh
+
+# Docker images used on Dockerfiles 2019-07-12
```

Comment:
we should start thinking to cache some of those images it will improve the reliability and the speed of build

Reply:
@kuisathaverat agree, we may want to ask for the status of https://github.com/elastic/infra/issues/8359

Reply:
it is not needed, this script pull the images on the immutable worker on build time, immutable workers packer images are built every 1-2 days, this means that when you launch an immutable worker the image is in the disk so docker does not make the pull. We are using this on APM for months and we did not have any more timeout issues.